opentelemetry-semantic-conventions/src/opentelemetry/semconv/schemas.py

```diff
@@ -0,0 +1,4 @@
+class Schemas(Enum):
```

Comment:
This is replacing the previously manually defined [SCHEMA_URL](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/__init__.py#L23) correct?

Reply:
yes, this should be a list now

Reply:
For now we won't have an automatic way to generate it/update the list and it would be a manual step. 
Unfortunately it can't be done with jinja templates.

We'd probably need to add a check into generate.sh that validates presence on new version in this file.

Reply:
added check to generate.sh